app/javascript/src/components/Dashboard/Notes/Note/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Typography, Dropdown } from "neetoui";
+
+const Header = ({ title }) => (
```

Comment:
Why do we need to abstract this into a separate component? Go through the Slack thread.

Reply:
Removed the separate components from Note.